codegen-luau/runtime/runtime.lua

```diff
@@ -85,6 +91,20 @@ do
 		return -num
 	end
 
+	function min.num(a, b)
```

Comment:
On second thought, it might be better to write these functions as...
```lua
if b == b then
    return math_func(a, b)
else
    return b
end
```
As it seems that's the style in which I wrote the other functions.